tests/nvme_test.py

```diff
@@ -17,6 +17,9 @@ def setUpClass(cls):
             raise unittest.SkipTest("nvme executable (nvme-cli package) not found in $PATH, skipping.")
         if not find_executable("nvmetcli"):
             raise unittest.SkipTest("nvmetcli executable not found in $PATH, skipping.")
+        ret, out, err = run_command("modprobe nvme-fabrics")
```

Comment:
It might be useful to create some helper "modprobe" function in utils if we are going to use this at more places.